### DISCUSSION TITLE
Problem with house number

### DISCUSSION THREAD

**Discussion by Pr3mut05** [mampfes/hacs_waste_collection_schedule#2317] [General]
Hello

When I try to add an entry, I get an error message

_The source returned an invalid response: "hnr expected, available: ['1', '3', '5', '7', '8 ,10,12,14,16,18,20,22,24', '11', '13', '15', '17', '19', '21', '25', '31', '32 ,34,34A', '33', '36', '38', '38 A', '40', '42', '44']". Please check the arguments and try again._

And this even though I am using one of the entries.
8 ,10,12,14,16,18,20,22,24
Is it because of the space?

![CleanShot 2024-07-20 at 23 27 05@2x](https://github.com/user-attachments/assets/8a0f1186-b629-425a-9813-eadedfbb816b)

**Comment by 5ila5** [comment]
There seem to be an issue that this source (and potentially otheres), that accept optional string and integer inputs currently only supports integer values. So hnr currently only allows you to input integers in the form. I fixed this behaviour in #2319, 
which will be part of the next release or is available now as master (git) version (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)


You could use one of your neighbours integer hnr for now if they have the same schedule.
